modules/json/src/main/scala/gem/json.scala

```diff
@@ -22,18 +23,34 @@ import scala.collection.immutable.TreeMap
 package object json {
 
   private implicit class MoreAngleOps(a: Angle) {
+    private def moveRight(n: Int): BigDecimal =
+      new java.math.BigDecimal(a.toMicroarcseconds).movePointRight(n)
```

Comment:
What's a `movePointRight`? scaling?

Reply:
Yes, fixed point math. Multiplication by multiple of 10 with no rounding error.